### PR TITLE
Try to speed up integration tests 2

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/browser/chromium/Chromium70.kt [hunk 1]
```diff
@@ -15,6 +15,18 @@ class Chromium70 : Browser {
      * Installs chromium 70 with a compatible chromedriver.
      */
     override fun install(ssh: SshConnection) {
+        ssh.execute("echo 'deb mirror://mirrors.ubuntu.com/mirrors.txt bionic main restricted universe multiverse' > /etc/apt/sources.list")
+        ssh.execute("echo 'deb mirror://mirrors.ubuntu.com/mirrors.txt bionic-updates main restricted universe multiverse' >> /etc/apt/sources.list")
+        ssh.execute("echo 'deb mirror://mirrors.ubuntu.com/mirrors.txt bionic-backports main restricted universe multiverse' >> /etc/apt/sources.list")
+        ssh.execute("echo 'deb mirror://mirrors.ubuntu.com/mirrors.txt bionic-security main restricted universe multiverse' >> /etc/apt/sources.list")
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/browser/chromium/Chromium70.kt:21]: First run it went to 20 minutes. The second one went to 13.
Perhaps it's Azure <-> Ubuntu mirrors slowness after all.
I'll port this ad hoc fix update to `ssh-ubuntu`.

Comment (dagguh) [reply src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/browser/chromium/Chromium70.kt]: Removing it and the mirrors from `ssh-ubuntu:0.2.1` helped

**Comment by dagguh** [comment]
@mzyromski-atlassian this PR is crucial. Without it, GHA goes to 60-90 minutes and often fails with timeouts.
With CI unblocked, I could finally bring other PRs to mergability

File: build.gradle.kts [hunk 2]
```diff
@@ -64,7 +64,7 @@ dependencies {
     testCompile("com.atlassian.performance.tools:jira-software-actions:[1.0.0,2.0.0)")
     testCompile("org.hamcrest:hamcrest-library:1.3")
     testCompile("org.assertj:assertj-core:3.11.1")
-    testCompile("com.atlassian.performance.tools:ssh-ubuntu:0.2.2")
+    testCompile("com.atlassian.performance.tools:ssh-ubuntu:0.2.0")
```
Comment (mzyromski-atlassian) [inline build.gradle.kts:67]: I'm trying to understand this. Does it mean `0.2.2` needs fixing?

Comment (dagguh) [reply build.gradle.kts:67]: Yes, 0.2.2 is bugged on GHA

Comment (dagguh) [reply build.gradle.kts:67]: Raised https://ecosystem.atlassian.net/browse/JPERF-723

**Review by mzyromski-atlassian — APPROVED** [review]

**Pull request merged**